src/components/Header_type_1.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { routeTo } from 'redux-router-kit';
+import { getTranslator } from '../reducers/translation';
+import { RoundHeaderButton } from './Buttons';
+
+import styles from './Header_type_1.scss';
+import common from '../styles/common.scss';
+
+const HeaderInitialComponent = ({ i18n, targetTo }) =>
+  (<header>
+    <div className={common.widthLimiter}>
+      <div className={styles.header}>
+        <Grid fluid>
+          <Row>
+            <Col lg={1} className={styles.logoContainer}>
+              <div className={styles.logo} />
+            </Col>
+            <Col lg={8} className={styles.mainArea}>
+              <div className={styles.title}>
+                {i18n('Neufund Platform')}
+              </div>
+              <div className={styles.status}>
+                {i18n('Log in')}
+              </div>
+            </Col>
+            <Col lg={3} className={styles.createButton}>
+              <RoundHeaderButton
+                onClick={(event) => {
```

Comment:
@Recmo Here I'm wondering about correct way to implement it. I'm hard coding `/register` address into component which is not best idea.  
We already have [PredefinedLinks](https://github.com/Neufund/generic-ico/blob/master/src/components/PredefinedLinks.js) component which renders <a> elements. Still surrounding <button> with <a> element is mistake. 

So now I ended in having `/register` path in two places. Should I abstract those targetTo to calls and reuse it across Header and PredefinedLinks?